docs: rephrase all sections (#191)
Open-Attestation/documentation-website: website/docs/docs-section/how-does-it-work/issuance-identity.md

---
id: issuance-identity
title: Issuance Identity
sidebar_label: Issuance Identity
---
import SupportedNetworks from "/src/reusables/_supported-networks.mdx";

OpenAttestation uses the Domain Name System (DNS) as the method to verify the issuer's identity. A one-line summary of the DNS system can be "a Phone book for the Internet". Its primary purpose is to resolve human readable names such as "google.com" or "openattestation.com" etc. to a set of records. IP addresses, the most common DNS records, make it possible for network routing to operate over the Internet.

> OpenAttestation uses records in the TXT format, so that it can store textual data. The textual data indicates the document store that the domain administrator trusts.

Reviewer: ```suggestion
OpenAttestation uses records in the TXT format which store textual data. The textual data indicate the document stores that the domain administrator trust.
```

Author: updated
